playground/frontend/lib/pages/embedded_playground/components/embedded_appbar_title.dart

```diff
@@ -53,20 +57,18 @@ class EmbeddedAppBarTitle extends StatelessWidget {
             },
             runCode: () {
               final stopwatch = Stopwatch()..start();
-              final exampleName = getAnalyticsExampleName(
-                state.selectedExample,
-                state.isExampleChanged,
-                state.sdk,
-              );
+              final exampleName = getAnalyticsExampleName(state);
+              final analyticsService = AnalyticsService.get(context);
+
               state.runCode(
                 onFinish: () {
-                  AnalyticsService.get(context).trackRunTimeEvent(
```

Comment:
This had been possessing the same risk of invalidated `context` as the bug above.